### PR TITLE
Per tab common dialog

### PR DESCRIPTION
**Pull request by piroor**
This does #2.

### CODE REVIEW

File: modules/main.js [hunk 1]
```diff
@@ -83,6 +85,9 @@ function handleWindow(aWindow)
     return;
   } else {
     log("generalWindow");
+    if (doc.documentElement.getAttribute("windowtype") == TYPE_BROWSER) {
```
Comment (myokoym) [inline modules/main.js]: Is there a reason to use `==`? Can we use `===`?

File: modules/main.js [hunk 2]
```diff
@@ -186,17 +203,17 @@ function processAction(aWindow, aAction)
   var value = actions[2];
   switch (action) {
   case 'accept':
-    doc.documentElement.acceptDialog();
+    commonDialog.onButton0();
```
Comment (myokoym) [inline modules/main.js]: `commonDialog.onButton0()` doesn't work without tabmodaldialog on my environment.
I think it should be replaced with `commonDialog.ui.button0.click()`.

File: modules/main.js [hunk 3]
```diff
@@ -186,17 +203,17 @@ function processAction(aWindow, aAction)
...
     log("accept");
     return;
   case 'cancel':
-    doc.documentElement.cancelDialog();
+    commonDialog.onButton1();
```
Comment (myokoym) [inline modules/main.js]: Can we use `commonDialog.ui.button1.click()`?

**Comment by piroor** [comment]
Thank you for feedback. I've fixed those careless mistakes.

**Comment by myokoym** [comment]
I've merged it. Thank you!

**Comment by piroor** [comment]
Thanks!

**Pull request merged**